R/setup_forested.R

```diff
@@ -0,0 +1,129 @@
+library(sf)
+library(tidymodels)
+library(spatialsample)
+library(tidysdm)
+library(forested)
+
+# ------------------------------------------------------------------------------
+
+tidymodels_prefer()
+theme_set(theme_bw())
+options(pillar.advice = FALSE, pillar.min_title_chars = Inf)
+
+# ------------------------------------------------------------------------------
+
+# These computations take a while so we do them in batch mode then load the 
+# results in later chapters
+
+# ------------------------------------------------------------------------------
+# We remove some of the existing algorithmic data already there. 
+
+for_analysis <-
+  forested %>%
+  rename(class = forested) %>%
+  select(-tree_no_tree, -land_type, -canopy_cover) 
```

Comment:
These are algorithmically derived features. Building on models with them seems a little redundant. Also, we want to make the problem a little more difficult.